### PR TITLE
Support the new label selector syntax and add the 'does not exist' op…

### PR DESCRIPTION
**Pull request by jwforres**
…tion to the label filter

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett

File: labelSelector.js [hunk 1]
```diff
@@ -12,15 +12,32 @@ function LabelSelector(selector, emptySelectsAll) {
   // TODO - currently k8s only returns key: value
```
Comment (spadgett) [inline labelSelector.js]: Can we remove the TODO comment?

**Comment by jwforres** [comment]
updated comment

**Comment by spadgett** [comment]
LGTM

**Pull request merged**